prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,19 @@
+var languages = ['html',
+'css','javascript','react','node','postgres']
+
+console.log('languages array', languages)
+
+var firstElement = languages
+
+ console.log('the first element of the array', languages[0])
```

Comment:
Here is where you would pass your `firstElement` variable.

```js
console.log('the first element of the array', firstElement);
```

you should update the rest of your variables like that